pybloqs/block/image.py

```diff
@@ -30,17 +31,43 @@
 _PLOT_DPI = 100
 
 
+@contextmanager
+def plotting_format(plot_format=None, dpi=None):
```

Comment:
It would be nice to be consistent in the naming of the regular and contextmanager versions.

Maybe:
```python
def plot_format(...
def set_plot_format(...
```